### DISCUSSION TITLE
force constant clock command

### DISCUSSION THREAD

**Discussion by nebutecharethebest** [NebuTech/NBMiner#488] [Ideas]
i need a command, -force constant coreclock
is this possible to be added in next patch?
i wasnt able to make it work through json file, cant find an example, then, it will be more simple with a command

for any reason my msi afterburner changes the clock even if it is locked.

**Comment by pauloalexandregoncalves** [comment]
it was very usefull to include command line arguments to define parameter for the GPU in windows, for instance:
Core Clock
Core Voltage
Memory Controller Voltage
Memory Clock
Memory Voltage

with this, no need to use msi afterburner ou other softwares and automation is more easy todo.
